### ISSUE TITLE
Customize sensors on screen

### ISSUE THREAD

**Issue by puspis** [CrazyDude1994/android-taranis-smartport-telemetry#25]
Hello.
It would be possible to customize the telemetry sensors Alt, GAlt, Hdg, etc. shown on screen and also set up the number of decimals for some sensors such as VFAS, Alt or Distance (for example 10.3km instead of 10372m)
Thanks

**Comment by CrazyDude1994** [comment]
That's what I'm working on right now. I think make it like opentx sensor scanner thing

**Comment by DiscoMan18** [comment]
And also would be very nice to display current GPS coordinates of the plane along with other telemetry data. And in case if the telemetry data stream suddenly stops (due to crashed plane, for example) maintain last good gps coordinates on the screen. This is very useful when you lost your plane.

**Comment by CrazyDude1994** [comment]
@DiscoMan18 for this purpose there is Replay log function. You can just use this to find latest known location

**Comment by DiscoMan18** [comment]
Yes, but as far as I know it's only shown as a position on the map. And I'm talking about having the last good GPS coords in numeric form, just to be able to put them into Google Maps during search and rescue mission ;)
